tests/integration/aof.tcl

```diff
@@ -295,4 +295,29 @@ tags {"aof"} {
             assert_equal $before $after
         }
     }
+
+    ## Test that the server exits when the AOF contains a unknown command
+    create_aof {
+        append_to_aof [formatCommand set foo hello]
+        append_to_aof [formatCommand bla foo hello]
+        append_to_aof [formatCommand set foo hello]
+    }
+
+    start_server_aof [list dir $server_path aof-load-truncated yes] {
+        test "Unknown command: Server should have logged an error" {
+            set pattern "*Unknown command 'bla' reading the append only file*"
+            set retry 10
+            while {$retry} {
+                set result [exec tail -1 < [dict get $srv stdout]]
+                if {[string match $pattern $result]} {
+                    break
+                }
+                incr retry -1
+                after 1000
+            }
+            if {$retry == 0} {
+                error "assertion:expected error not found on config file"
+            }
```

Comment:
i suppose you copied that from another test in this file, but we now have `wait_for_log_messages`.
so i'm not sure if we wanna do massive cleanup in this file, but let's at least start using that in new code.

Reply:
since we are using start_server_aof, the server that started doesn't get into the list of existing servers, and because wait_for_log_messages get srv_idx as input parameter it wouldn't work.